cumulusci/cli/cci.py

```diff
@@ -368,6 +330,23 @@ def service():
     pass
 
 
+@cli.group("error")
+def error():
+    """
+    If you'd like to dig into an error more yourself,
+    you can get the last few lines of context about it
+    from `cci error info`.
+
+    If you'd like to submit it to a developer for conversation,
+    you can use the `cci error gist` command. Just make sure
+    that your GitHub access token has the 'create gist' scope.
+
+    For more information on working with errors in CumulusCI visit:
+    https://cumulusci.readthedocs.io/en/latest/features.html#working-with-errors
+    """
+    pass
```

Comment:
Python trivia: the docstring does the same thing as pass, so you actually don't need it here (although some might argue that it helps clarify the intent). The ellipsis operator also does a passable job as pass, but I'm sure someone would complain if you used it that way.

Reply:
TIL!

Reply:
👨‍🎓

Reply:
Guido's take on the difference between `pass`, `...`, and `None`: https://twitter.com/llanga/status/1216312568131792896